zipline/utils/cache.py

```diff
@@ -334,7 +353,11 @@ def mkdir(self, *path_parts):
             The parts of the path after the working directory.
         """
         path = self.getpath(*path_parts)
-        os.mkdir(path)
+        try:
```

Comment:
Ah, I see. There's actually no possible way for the directory to exist because it's within a temp directory...I think. I could be wrong. 

But yeah you're right about it not failing, that was for catching the errors in testing.